server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,225 @@
+/* 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+import java.util.List; 
+ 
+public class Ad {
+  // fields
+  public String id; 
+  public String advertiser;  
+  public String startDate; // isostring format
+  public String endDate; //isostring
+  public long impressionsMin; 
+  public long impressionsMax; 
+  public boolean isTargetingAge; 
+  public List<String> genderTargets; 
```

Comment:
It would be helpful to add more comments for the expected format for each of these, or what some of them are.   You could do this with javadocs on the getters+setters.